libcontainer/specconv/spec_linux.go

```diff
@@ -301,6 +301,9 @@ func CreateLibcontainerConfig(opts *CreateOpts) (*configs.Config, error) {
 		}
 		if spec.Linux.IntelRdt != nil {
 			config.IntelRdt = &configs.IntelRdt{}
+			if spec.Linux.IntelRdt.ClosID != "" {
+				config.IntelRdt.ClosID = spec.Linux.IntelRdt.ClosID
+			}
```

Comment:
So, this makes no sense, as both variables are strings, the default value is `""` and so `if` is totally redundant.

I just found this by looking into intelrtd code; will open a small fix PR soon.

Reply:
Resulted in not-so-small PR although I think it makes sense to have that before this one.

@marquiz PTAL  https://github.com/opencontainers/runc/pull/3041 🙏🏻

Reply:
Resolved by rebase